src/jobservice/job/impl/gc/garbage_collection.go

```diff
@@ -442,7 +442,11 @@ func (gc *GarbageCollector) removeUntaggedBlobs(ctx job.Context) {
 				},
 				PageNumber: 1,
 				PageSize:   int64(ps),
-				Sorting:    "id",
+				Sorts: []*q.Sort{
+					{
+						Key: "id",
+					},
+				},
```

Comment:
How about making the `parseSorting` in the q pkg be public and change these to `Sorts: q.ParseSorting("id")`?

Maybe I prefer `q.NewSorts("id")` than `q.ParseSorting("id")`.